src/components/TableCard.tsx

```diff
@@ -0,0 +1,102 @@
+import { ReactNode, useMemo } from "react";
+import { useHover } from "react-aria";
+import { Css, Palette } from "src/Css";
+import { ButtonMenu, MenuItem } from "./ButtonMenu";
+import { Tag, TagType } from "./Tag";
+
+export type TableCardType = "card" | "list";
+
+type CardTag = {
+  text: string;
+  type?: TagType;
+};
+
+export interface TableCardProps {
+  title: string;
+  subtitle: string;
+  metadata: ReactNode;
```

Comment:
Thought: I wonder if there will be any use cases where we'll want to be able to just have a card with a title/subtitle/image. I haven't seen it in any designs but we could make the prop optional. That would also be an easy change down the line so fine to leave it as is now